README.md

```diff
@@ -97,13 +97,24 @@ To enable this, add the following to your /etc/docker/daemon.json config:
 ```
 
 ### Local Setup
-Now, compile the main server using Maven 3.5 with Java 8:
+Now, compile the main server using Maven with Java:
 
     mvn install
 
 This will create `shesmu-server/target/shesmu.jar`. If you require any
-additional plugins (described below), compile them and collect all the JARs in
-a directory on your server in `/srv/shesmu` or a path of your choosing.
+additional plugins (described below), compile them and collect all the JARs and
+their dependencies in a directory on your server in `/srv/shesmu` or a path of
+your choosing. The complete set of plugins and dependencies can be copied into
+a `jars` directory inside the build directory by doing:
+
+```
+test -d jars || mkdir jars
+cd jars
+ln -sf ../install-pom.xml pom.xml
+rm -rf *.jar
+mvn -DVERSION=$(cd ..; mvn help:evaluate -Dexpression=project.version -q -DforceStdout) dependency:copy-dependencies
+cd ..
```

Comment:
should this include copying the jars to /srv/shesmu/jars?

Reply:
The advice here is really not great installation advice. It doesn't reflect the release process OICR is using and it's not really appropriate for anyone who wants to follow the builds produced by OICR, so I'm not sure what the right thing to say is.

Reply:
I just mean that lines 106-108 tell you to put the jars in /srv/shesmu/jars but then this code block does not appear to do that despite the claim on line 108 that it will?

Reply:
Okay. How's that?

Reply:
I think a fix was somehow undone here